_data/people.yml

```diff
@@ -330,3 +339,4 @@
    role: alumni
    linkedin: william-wang-a445b4193
    pic: wwang.jpg
+>>>>>>> 2e45997d69a8389896705e633dc7f498d439376c
```

Comment:
I see there are still conflicts. I recommend you close this PR, you `pull --rebase` the repo and open a new one. @calebescobedo